Backend/sprintone/src/main/java/com/project/omega/enums/RoleEnum.java

```diff
@@ -0,0 +1,5 @@
+package com.project.omega.enums;
+
+public enum RoleEnum {
```

Comment:
Roles need to be stored in the database as each of our clients will have the ability to create custom roles. The only roles in there that cannot be changed are `Admins` (which are the admins of our clients), and `Super Admins` (which are admins in our team)

Reply:
This file should be there, follow the code I committed last night. I have asked Alex to close the existing pull request and follow one true branch.